### ISSUE TITLE
package gets removed by npm

### ISSUE THREAD

**Issue by voice-technologies** [500tech/angular-tree-component#616]
I installed angular-tree-component using:

npm install angular-tree-component --save

Everything worked fine. But now, whenever I install some other package (e.g. a datepicker) with npm, angular-tree-component is automatically removed. If I then reinstall angular-tree-component, the other package I just installed is automatically removed!

What is going on, and how do I get around this? I'm not a npm expert in any way, so I may be doing something incredibly stupid, but this kind of thing has never happened with any other packages that I have installed.

Basically, I need both a tree and a datepicker component on the same page: is this even possible?

**Comment by tommy-hansen** [comment]
https://github.com/npm/npm/issues/17379? Try googling "npm install removes packages".

**Comment by voice-technologies** [comment]
Yes, I had already done searches using similar terms and found nothing helpful. My point is that whatever else I try to install, it is always and only angular-tree-component (a-t-c) that gets kicked out, which is what makes me think the post referred to is addressing a different issue. I have several other angular-2+ packages installed (mostly 3rd party components) and none of them are affected in the same way. So it seems that there has to be something different about a-t-c. If I know what that is then I can start searching for a workaround. If you want to try and reproduce just npm install a-t-c followed by (say) angular4-datepicker. Then, having lost a-t-c, install it again and you will lose the datepicker. Btw, this is on a Windows 7 platform, for what difference that makes.

**Comment by voice-technologies** [comment]
Worked around it by using an old version of npm. But still not sure why the problems only started when I installed angular-tree-component.
